src/form/ChestAPI/session/network/handler/PlayerNetworkHandler.php

```
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace pocketmine\form\ChestAPI\session\network\handler;
+
+use Closure;
+use pocketmine\form\ChestAPI\session\network\NetworkStackLatencyEntry;
+
+interface PlayerNetworkHandler{
+
+	public function createNetworkStackLatencyEntry(Closure $then) : NetworkStackLatencyEntry;
```

PHPStan (2.1.15)

Method pocketmine\form\ChestAPI\session\network\handler\PlayerNetworkHandler::createNetworkStackLatencyEntry() has parameter $then with no signature specified for Closure.


(missingType.callable)